django2pydantic/handlers/numbers.py

```
@@ -1,154 +1,119 @@
-        return min(connection.ops.integer_field_range("IntegerField")[1], super().le)
+        validator_max = super().le
+        db_max = connection.ops.integer_field_range(self.field().__name__)[1]
+        return min(db_max, validator_max) if validator_max is not None else db_max
 
     @override
     def get_pydantic_type_raw(self) -> type[int]:
         return int
 
+    # TODO(phuongfi91): This might cause issue with schemathesis test
+    #  https://github.com/NextGenContributions/django-ninja-crudl/issues/35
+    # examples = [-9223372036854775808, 9223372036854775807]
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)